sidebars.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
             'functions/run-function/run-java-function',
             'functions/run-function/run-python-function',
             'functions/run-function/run-go-function',
+            'functions/run-function/run-stateful-function',
```

Comment:
double-check: do we need a comma for the last item in `sidebar.js`? (in Pulsar doc, no)

Reply:
It seems the doc can be correctly displayed whether there is a comma or there is no comma for the last item in the sidebar.js. Anyway, I remove the comma.